docs/storing-querying/querying-data/index.md

```diff
@@ -265,7 +308,7 @@ select
     ...
 from
     unique_events
-where 
+where
```

Comment:
We could captialize these too.  Mentioning it because you've already done some capitalization fixes elsewhere in this PR.